src/semaphore/worker/functions/send_message.py

```diff
@@ -0,0 +1,22 @@
+"""A proof-of-concept worker function."""
+
+from __future__ import annotations
+
+from typing import Any, Dict
+
+import structlog
+from safir.slack.blockkit import SlackMessage
+from safir.slack.webhook import SlackWebhookClient
+
+from semaphore.config import config
+
+
+async def send_message(ctx: Dict[Any, Any], message: str) -> None:
+    logger = ctx["logger"].bind(task="send_message")
+    logger.info("Running send_message")
+
+    logger = structlog.get_logger(__name__)
+    client = SlackWebhookClient(config.slack_webhook_url, "App Name", logger)
```

Comment:
```suggestion
    client = SlackWebhookClient(config.slack_webhook_url.get_secret_value(), "Semaphore", logger)
```

See https://docs.pydantic.dev/usage/types/#secret-types for using Pydantic secrets.

Reply:
You might set up the SlackWebhookClient in the worker startup function and stash it in the context dictionary, `ctx` so that you don't have to set up/configure it in the task function.